src/components/modules/ModuleNavigationAlt.tsx

```diff
@@ -0,0 +1,156 @@
+import { useIOTheme } from "../../context";
+import {
+  IOListItemVisualParams,
+  IOSelectionListItemVisualParams,
+  IOSpacer,
+  IOVisualCostants
+} from "../../core";
+import { useIOFontDynamicScale } from "../../utils/accessibility";
+import { WithTestID } from "../../utils/types";
+import { Badge } from "../badge";
+import { IOIcons, Icon } from "../icons";
+import { HStack, VStack } from "../layout";
+import { IOSkeleton } from "../skeleton";
+import { BodySmall, LabelMini } from "../typography";
+import { ModuleStatic } from "./ModuleStatic";
+import {
+  PressableModuleBase,
+  PressableModuleBaseProps
+} from "./PressableModuleBase";
+
+type LoadingProps = {
+  isLoading: true;
+  loadingAccessibilityLabel?: string;
+};
+
+type ImageProps =
+  | { icon: IOIcons; image?: never }
+  | { icon?: never; image: ImageURISource | ImageSourcePropType };
+
+type BaseProps = {
+  isLoading?: false;
+  title: string;
+  subtitle?: string;
+  badge?: Badge;
+} & ImageProps &
+  PressableModuleBaseProps;
+
+type ModuleNavigationAltProps = LoadingProps | BaseProps;
+
+export const ModuleNavigationAlt = (
+  props: WithTestID<ModuleNavigationAltProps>
+) => {
+  const theme = useIOTheme();
+  const { hugeFontEnabled } = useIOFontDynamicScale();
+
+  if (props.isLoading) {
+    return (
+      <ModuleNavigationAltSkeleton
+        loadingAccessibilityLabel={props.loadingAccessibilityLabel}
+      />
+    );
+  }
+
+  const { icon, image, title, subtitle, onPress, badge, ...pressableProps } =
+    props;
+
+  const iconComponent = icon && !hugeFontEnabled && (
+    <Icon name={icon} size={32} color="blueIO-500" />
+  );
+
+  const imageComponent = image && (
+    <Image
+      source={image}
+      style={styles.image}
+      accessibilityIgnoresInvertColors={true}
+    />
+  );
+
+  return (
+    <PressableModuleBase {...pressableProps} onPress={onPress}>
+      <HStack space={8} style={{ alignItems: "center" }}>
+        <HStack
+          space={IOVisualCostants.iconMargin as IOSpacer}
+          style={{ alignItems: "center", flexGrow: 1, flexShrink: 1 }}
+        >
+          {iconComponent ?? imageComponent}
+
+          <View style={{ flexShrink: 1 }}>
+            <View
+              style={{
+                display: "flex",
+                alignItems: "flex-start",
+                marginBottom: 8
+              }}
+            >
+              {badge ? <Badge {...badge} /> : null}
+            </View>
+            <BodySmall
+              color={theme["interactiveElem-default"]}
+              weight="Semibold"
+              numberOfLines={2}
+              lineBreakMode="middle"
+              style={{ flexShrink: 1 }}
+            >
+              {title}
+            </BodySmall>
+            {subtitle && (
+              <LabelMini weight="Regular" color={theme["textBody-tertiary"]}>
+                {subtitle}
+              </LabelMini>
+            )}
+          </View>
+        </HStack>
+        {onPress ? (
+          <Icon
```

Comment:
What do you think about making this icon customizable? We could consider passing it from outside, so that it can be customized in case of different future versions.